scripts/servers_test.py

```diff
@@ -579,21 +608,21 @@ def mock_os_path_exists(path):
         self.assertEqual(popen_calls[0].kwargs, {'shell': True})
         self.assertEqual(mock_os_remove.times_called, 1)
 
-    def test_managed_web_browser_on_linux_os(self):
+    def test_managed_web_browser_on_linux_os(self) -> None:
         popen_calls = self.exit_stack.enter_context(self.swap_popen())
         self.exit_stack.enter_context(self.swap(common, 'OS_NAME', 'Linux'))
         self.exit_stack.enter_context(self.swap_to_always_return(
             os, 'listdir', value=[]))
 
         managed_web_browser = servers.create_managed_web_browser(123)
-        self.assertIsNotNone(managed_web_browser)
+        assert managed_web_browser is not None
```

Comment:
Why this change? We usually prefer to use `self.assertIsNotNone` for testing. Ditto below

Reply:
Here `managed_web_browser` is of `Optional[ContextManager]` type and below in line 620, `exit_stack.enter_context` method can only accept ContextManager type not None. So, to narrow down the type by keeping the assertion in place I did this change.

Basically, `self.assertIsNotNone` only checks for None but it does not narrow the type for type checking. Whereas, `assert is not None` check for None values and also narrow down type from Optional types to non-optional types.

Reply:
OK, in that case, can you do both? i.e. the code would look like this:

```python
        self.assertIsNotNone(managed_web_browser)
        assert managed_web_browser is not None
```

The first line is for the test, while the second line is for type checking. The reason to keep the `self.assertIsNotNone` is twofold. First, by convention, the assertions we are making in the tests use the self.assert... methods, so by using just a plain `assert`, you're implying that the assertion isn't part of what the test is checking for. Second, `unittest` gives us more informative debugging output when you use its methods than if you just use a plain `assert`. See https://stackoverflow.com/a/45947566 for more explanation

Reply:
Thanks for the information!
```python
        self.assertIsNotNone(managed_web_browser)
```
is added.